quickwit/quickwit-ingest/src/ingest_v2/mrecord.rs

```diff
@@ -37,14 +38,10 @@ const DOC_HEADER_V0: &[u8; MRECORD_HEADER_LEN] = &[HeaderVersion::V0 as u8, 0];
 /// `Commit` header v0 composed of the header version and the `Commit = 1` record type.
 const COMMIT_HEADER_V0: &[u8; MRECORD_HEADER_LEN] = &[HeaderVersion::V0 as u8, 1];
 
-/// `Eof` header v0 composed of the header version and the `Eof = 2` record type.
-const EOF_HEADER_V0: &[u8; MRECORD_HEADER_LEN] = &[HeaderVersion::V0 as u8, 2];
-
 #[derive(Debug, Clone, Eq, PartialEq)]
 pub enum MRecord {
     Doc(Bytes),
     Commit,
-    Eof,
     Unknown,
```

Comment:
```suggestion
    Unknown, 
```

Unknown is, I believe, an error here.
We can remove it and return an Option<Self> to decode.

Reply:
Yes, we can remove this variant.